internal/host/instructionmanager.go

```diff
@@ -48,6 +49,7 @@ type InstructionConfig struct {
 	FreeAddressesImage      string `envconfig:"FREE_ADDRESSES_IMAGE" default:"quay.io/ocpmetal/assisted-installer-agent:latest"`
 	DhcpLeaseAllocatorImage string `envconfig:"DHCP_LEASE_ALLOCATOR_IMAGE" default:"quay.io/ocpmetal/assisted-installer-agent:latest"`
 	SkipCertVerification    bool   `envconfig:"SKIP_CERT_VERIFICATION" default:"false"`
+	InstallationTimeout     uint   `envconfig:"INSTALLATION_TIMEOUT" default:""`
```

Comment:
default should be 0 no?

Reply:
this is equal to the zero value, it will initialize as 0

Reply:
I understand that empty string converts to but why initialize a uint with a string?